index.html

```diff
@@ -63,8 +63,447 @@ <h2 class="who-am-i">I’m a software developer</h2>
             
          
 
-        </section> 
+        </section>         
+        <div class="collapse-button">
+            <img src="asset/images/Disabled.png" alt="collapse project-button">
+        </div>          
+
+        <h2 class="works-heading"> MY RECENT WORKS</h2>            
+
```

Comment:
- [ ] All the `h2` should be under `section` tag.

Reply:
h2 tag is now under a section tag.